config/routes.rb

```diff
@@ -132,6 +132,7 @@ def external_redirect(path_prefix, target)
     get "/consultations/:consultation_id/outcome/:id" => "_#_", as: "consultation_outcome_html_attachment"
     get "/consultations/:consultation_id/public-feedback/:id" => "_#_", as: "consultation_public_feedback_html_attachment"
 
+    # TODO: can we remove?
```

Comment:
Could this comment and the commit message be updated to explain why it is still there, e.g. for `/get-involved` we say:
```
# Controller removed. Whitehall frontend no longer serves these
# pages however the route is needed to generate path and urlgenerate path and url
# helper methods.
```